### ISSUE TITLE
Look into ToS & related legalish stuff

### ISSUE THREAD

**Issue by Throne3d** [Marri/glowfic#602]
- Get a ToS? Notify users when it's updated, *perhaps* block writing on the site until it's agreed to but perhaps just have a term that we'll make a reasonable effort to notify of changes and then do it by email / in-site notification?
- Check our handling of private data? (UK has Data Protection Act, EU has similar laws – GDPR? – do we need to alter anything to ensure we comply?)
- Cookie banner?

**Comment by Marri** [comment]
More things we may need:
- Privacy policy, goes with ToS. Require agreement at sign up, block-to-backfill-agreement for existing users.
- Maaaaybe encrypt email addresses?
- There's a Right to Erasure thing that looks interesting under the GDPR, but that might be as simple as "we allow you to request a dev/mod detaches your email address from your account", it's not like an email address is required functionality. (UI required, sure, but nothing would break except notifications.)

**Comment by Throne3d** [comment]
To comply with the GDPR, we need to have several things. I can probably come up with a more exhaustive list than this, but:

- A list of data we gather, and why
- The whole privacy policy written in plain English
- Who they should contact in the event we have some erroneous data gathered about them
- Who they can appeal to should we fail to respond promptly to requests like this
- Right to erasure (need to figure out what's up with that)
- Anything third-party (like NewRelic?) might need to be consented to

We should also do this ASAP since GDPR went active back in May, iirc.

**Comment by Marri** [comment]
Leaving open for now despite #919 existing because I have not yet confirmed GDPR compliance

**Comment by l1n** [comment]
This is marked crit, what is needed to actually confirm GDPR compliance?

**Comment by Throne3d** [comment]
I think we're compliant enough at this point - we have clear consent + clear explanations of how we use data in our privacy policy (linked from the TOS) and we have a feature to be able to delete a user account on request. https://www.glowfic.com/privacy

AIUI these are our main obligations
